### DISCUSSION TITLE
How to change cookies secure setting?

### DISCUSSION THREAD

**Discussion by fengerzh** [chimurai/http-proxy-middleware#533] [Q&A]
I can use `cookieDomainRewrite` and `cookiePathRewrite` to rewrite cookies, but there are more other settings in cookies, such as `secure` and `SameSite`, as server is `https://www.something.com`, it set cookies with `secure`, but my `localhost` is `http://localhost:8080`, so if write cookies with `secure` switch on, Chrome will refuse to set.

Can this flag be changed by this middleware? Thanks!

**Comment by psi-4ward** [comment]
Yes, an option-flag would be convenient.

But for know a little hack:
```
    proxy: {
      '/login': {
        onProxyRes(proxyRes, req, res) {
          if (proxyRes.headers['set-cookie']) {
            proxyRes.headers['set-cookie'][0] = proxyRes.headers['set-cookie'][0].replace('; Secure', '');
          }
        },
      },
```
